### PR TITLE
Add HomeWorkOOP_1_3

### PR DESCRIPTION
**Pull request by AndriiNazaruk**
Add Home Work OOP 1_3 without nUnit,
I will try to add nUnit test cases on Monday

### CODE REVIEW

**Review by melnyck — DISMISSED** [review]

File: Projects/Team3/Andrii Nazaruk/HomeWork_OOP_1_3/Food.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            try
+            {
+                Console.WriteLine("Enter Product Name (String)");
+                NameOfProduct = Console.ReadLine();
+                Console.WriteLine("Enter Purchase Price (Double) ");
+                PurchasePrice = Convert.ToDouble(Console.ReadLine());
+                Console.WriteLine("Enter Purchase Date (DateTime like '2010.1.6') ");
+                PurchaseDate = Convert.ToDateTime(Console.ReadLine());
+                Console.WriteLine("Enter Stock Balance (int) ");
+                StockBalance = Convert.ToInt32(Console.ReadLine());
```
Comment (romanukhaav) [inline Projects/Team3/Andrii Nazaruk/HomeWork_OOP_1_3/Food.cs]: please use TryParse() or try/catch

Comment (AndriiNazaruk) [reply Projects/Team3/Andrii Nazaruk/HomeWork_OOP_1_3/Food.cs]: All code in function on try/catch.Replace Convert to TryParse()?

File: Projects/Team3/Andrii Nazaruk/MyFirstNUnitTest/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        private IWebDriver driver;
+
+        [OneTimeSetUp]
+        public void BeforeAllMethods()
+        {
+            driver = new ChromeDriver();
+        
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+        }
+
```
Comment (romanukhaav) [inline Projects/Team3/Andrii Nazaruk/MyFirstNUnitTest/UnitTest1.cs:20]: please add driver.Quit()

Comment (AndriiNazaruk) [reply Projects/Team3/Andrii Nazaruk/MyFirstNUnitTest/UnitTest1.cs:20]: ok

**Review by romanukhaav — APPROVED** [review]

**Pull request merged**